src/ext/TagExtension/wixext/TagBinder.cs

```diff
@@ -198,11 +198,18 @@ private List<WixFileRow> CreateProductTagFiles(Output output)
 
         private static Version CreateFourPartVersion(string versionString)
         {
-            Version version = new Version(versionString);
-            return new Version(version.Major,
-                               -1 < version.Minor ? version.Minor : 0,
-                               -1 < version.Build ? version.Build : 0,
-                               -1 < version.Revision ? version.Revision : 0);
+            try
```

Comment:
Turns out the tag is generated twice. First time just to get the size about right and second time to actually get the tag fully populated with access to bind variables. Thus the version may start as "0.0" and be updated to bind values. So it works out okay.